spec/integration_spec.rb

```diff
@@ -14,12 +14,14 @@
     ENV.delete("JEKYLL_ENV")
     ENV["JEKYLL_ENV"] = "production"
     ENV["PAGES_ENV"] = "dotcom"
+    @original_log_level = Jekyll.logger.level
     Jekyll.logger.log_level = :error
     Jekyll::Commands::Build.process(config_defaults)
   end
   after(:each) do
     ENV.delete("PAGES_ENV")
     ENV["JEKYLL_ENV"] = "test"
+    Jekyll.logger.log_level = @original_log_level
```

Comment:
What do you think about moving this after and the before as well into the spec_helper.rb file? That way every spec can have the log level reset, so all they need to do is set a new log level as desired.

Reply:
@parkr I agree, that is a good idea. I'm pushing a commit to do this. Please let me know what you think.